example/roles/overlays/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
Probably makes more sense to name the role 'images' ? Since you're getting all the pxe/coreOS stuff too.